src/web/client/common/constants.ts

```diff
@@ -98,12 +98,13 @@ export enum queryParameters {
     ORG_URL = "orgurl",
     REGION = "region",
     ENV_ID = "envid",
-    GEO = "geo",
+    GEO = "geo", // User geo location
     ENABLE_MULTIFILE = "enablemultifile",
     WEBSITE_PREVIEW_URL = "websitepreviewurl",
     ENTITY = "entity",
     ENTITY_ID = "entityid",
-    REFERRER_SOURCE = "referrersource"
+    REFERRER_SOURCE = "referrersource",
+    ORG_GEO = 'orgGeo'
```

Comment:
with Artemis changes do we really need this change now? Lets not do both

Reply:
Make sense. Removed